### PR TITLE
docs(pr_template): various fixes and comment updates

### PR DESCRIPTION
**Pull request by wdhdev**
This PR aims to fix some grammar in the PR template and move some text to comments in order to reduce the visible length of the PR template when finished.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,27 +11,28 @@ https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change
...
 
 A recent PR using the current template is 
 https://github.com/publicsuffix/list/pull/1591, although 
 the organization and description were not as substantial 
 as desired, which required maintainers time to visit the 
-requestors website to further research. 
-Having more robust org/desc improves the PR processing 
-pace due to the extra cycles not lost to research.
+requestors website to further research.
+Having more robust responses improves the PR processing
```
Comment (dnsguru) [inline .github/pull_request_template.md:21]: We literally want more robust responses to both the organization and the description, which was the origin of the specific wording.  Absent explaining this we were getting really basic crap answers or cut-paste marketing smeg that did not articulate the rationale for inclusion in PSL

Comment (dnsguru) [reply .github/pull_request_template.md:21]: so maybe, please, leave this as ```org/desc```

Comment (wdhdev) [reply .github/pull_request_template.md]: https://github.com/publicsuffix/list/pull/2156/commits/bb05a8e667f631f4a41d3837285903fe57eec6a4

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -44,10 +46,12 @@ your domain(s) directly with that third-party, and it is inappropriate
 to submit entries to the PSL as a means to work around those limits or 
 restrictions.
 -->
-  * [ ] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
-    - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
-    - [Letsencrypt](https://letsencrypt.org/docs/rate-limits/)
-    - <UPDATE THIS LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! REMOVE THIS LINE!>
+
+<!-- MAKE SURE UPDATE THE FOLLOWING LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! -->
```
Comment (simon-friedberger) [inline .github/pull_request_template.md:50]: This was intentionally part of the list, to make it very obvious when people haven't at least modified the list to remove it.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -65,38 +69,38 @@ requesting party that they are not attempting to bypass rate limits through
...
+The ongoing trust of the PSL requires it to be free of outdated or problematic entries. In making this pull request, there is a commitment by the submitter that they are going to review and maintain their relevant section. By submitting an entry, the requestor acknowledges that their entry and section may be removed if the domain does not maintain the respective _psl entries in DNS, any domain(s) with-in their section fail to resolve in DNS, the domain does not get renewed, expires or is otherwise unreachable. The submitter further identifies that it is their responsibility to review their submitted section within the PSL, submitting updates or removals as their domain(s) may change over time. It is also the responsibility of the submitter to provide (and keep up to date) a reachable email address within the section, and to maintain that address as it may change over time, so that they receive notices.
```
Comment (simon-friedberger) [inline .github/pull_request_template.md:77]: "with-in"?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -176,23 +173,19 @@ this area to describe how and detail the manner in which you
 have first attempted to engage those third parties on the 
 matter.
 
-Please also include the numbers of any past Issue # or PR # 
+Please also referance any past issues or PRs
```
Comment (simon-friedberger) [inline .github/pull_request_template.md:176]: reference

**Pull request merged**